install/kubernetes/quick-install.yaml

```diff
@@ -280,6 +280,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  # To be used during HA deployment of cilium-operator in order
+  # to obtain lease lock for leader election.
+  - leases
+  verbs:
+  - '*'
```

Comment:
And here as well, we might need the same check as above on the K8s version?

Reply:
I agree, if there is any aspect here that involves additional extra work to support this operator HA feature on k8s 1.14 or below then I don't think it's worth adding that support to Cilium at this point.